src/framework/components/element/component.js

```diff
@@ -806,7 +755,11 @@ class ElementComponent extends Component {
         this._setWidth(wr - wl);
 
         // update position
-        p.x = (this._localAnchor.z - this._localAnchor.x) - value - (this._calculatedWidth * (1 - this._pivot.x));
+        p.x =
+            this._localAnchor.z -
+            this._localAnchor.x -
+            value -
+            this._calculatedWidth * (1 - this._pivot.x);
```

Comment:
I'm a bit unsure about all these Prettier edits. I mean, is it _really_ prettier? 🤔  That said, I don't feel super-strongly, so I'll just hit approve...

Reply:
Yea a lot of that im reverting its just for the new code ive added a lot of this i need to go back over and revert